.github/workflows/cypress.yml

```diff
@@ -0,0 +1,44 @@
+name: Cypress Tests
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  cypress-run:
+    name: Cypress Run
+    if: "!contains(github.event.head_commit.message, 'chore(release):')"
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [12.x, 14.x, 15.x]
+    env:
+      CYPRESS_BASE_URL: https://konviw.vercel.app/cpv
+#      CPV_CONFLUENCE_API_TOKEN: ${{ secrets.CPV_CONFLUENCE_API_TOKEN }}
+#      CPV_CONFLUENCE_API_USERNAME: ${{ secrets.CPV_CONFLUENCE_API_USERNAME }}
+#      CPV_CONFLUENCE_BASE_URL: ${{ secrets.CPV_CONFLUENCE_BASE_URL }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Setup Node
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+
+      - name: Cache dependencies
+        uses: actions/cache@v2
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Run Cypress Tests
+        run: npm run cypress:run
```

Comment:
Did you check the Cypress official GitHub action? https://docs.cypress.io/guides/continuous-integration/github-actions#Caching-Dependencies-and-Build-Artifacts

It would be best to use this, and also upload the artifact so that we can view the video and screenshots of the test run.

Reply:
I've updated the cypress step to use `cypress-io/github-action@v2` and also added artifacts for videos and snapshots.
![Screenshot 2022-02-21 at 22 55 04](https://user-images.githubusercontent.com/36277587/155030812-15bd29b8-a432-4470-b86c-db9bc415daae.png)